src/port1.0/portutil.tcl

```diff
@@ -3372,7 +3379,11 @@ proc _check_xcode_version {} {
             }
 
             # Check whether /usr/include and /usr/bin/make exist and tell users to install the command line tools, if they don't
-            if {${os.major} <= 17 && (![file isdirectory [file join $cltpath usr include]] || ![file executable  [file join $cltpath usr bin make]])} {
+            if {${os.major} <= 18 && (![file isdirectory [file join $cltpath usr include]] || ![file executable  [file join $cltpath usr bin make]])} {
+                if {${os.major} == 18 && [option configure.sdk_version] ne "" && [file tail [option configure.sdkroot]] ne "MacOSX[option configure.sdk_version].sdk"} {
+                    ui_warn "The macOS [option configure.sdk_version] SDK does not appear to be installed. Ports may not build correctly."
+                    ui_warn "You can install it as part of the Xcode Command Line Tools package by running `xcode-select --install'."
+                }
```

Comment:
The else blocks of the two if statement below this are still going to execute and print incorrect warnings.

Reply:
I can swap the xcode check to elseif or move this check into its own block that only deals with Mojave